### PR TITLE
Enable SampleCmpLevelZero tests and update the results

### PR DESCRIPTION
**Pull request by pmistryNV**

### CODE REVIEW

**Review by csyonghe — APPROVED** [review]

**Comment by pmistryNV** [comment]
@csyonghe kindly approve, I have added rest of the tests

**Comment by csyonghe** [comment]
We need to fix the CI failure before merging this.

File: tests/hlsl-intrinsic/texture/texture-intrinsics.slang [hunk 1]
```diff
@@ -59,7 +59,6 @@ void computeMain(int3 dispatchThreadID: SV_DispatchThreadID)
     //  NOTE: The "cpu" profile does not like these, so it's disabled for now
     val += t1D.SampleLevel(samplerState, u, 0, 1);
     val += t2D.SampleLevel(samplerState, float2(u, u), 0, int2(1, 1));
-    val += t3D.SampleLevel(samplerState, float3(u, u, u), 0, int3(1, 1, 1));
```
Comment (csyonghe) [inline tests/hlsl-intrinsic/texture/texture-intrinsics.slang:62]: Why do we need to remove this?

**Comment by pmistryNV** [comment]
@csyonghe please approve

File: tests/hlsl-intrinsic/texture/texture-intrinsics.slang (hunk 1 above)
Comment (pmistryNV) [reply tests/hlsl-intrinsic/texture/texture-intrinsics.slang:62]: It's a mistake let me add it back

**Review by csyonghe — DISMISSED** [review]

**Review by csyonghe — APPROVED** [review]

**Pull request merged**